stories/Forms.story.jsx

```diff
@@ -162,4 +72,70 @@ storiesOf("Forms", module)
       </PageLayout>
     ))
   )
-  .add("form with validation", withInfo()(() => <FormWithValidation />));
+  .add(
+    "form with validation",
+    withInfo()(() => (
+      <Validate rules={rules}>
+        {({
+          onSubmit,
+          onBlur,
+          onChange,
+          errors: { name, lastName, email, terms, newsletter }
```

Comment:
what does this do?

Reply:
its the same like if I would do const  `{ name, lastName, email, terms, newsletter } = errors` 3 lines later.